src/translations/v6/v6.ts

```diff
@@ -92,6 +96,101 @@ export const getV6Translations = (abstractSqlModel = v6AbstractSqlModel) =>
 			'is managed by-device': ['Cast', ['Null'], 'Integer'],
 			'logs channel': ['Cast', ['Null'], 'Short Text'],
 			'vpn address': ['Cast', ['Null'], 'Short Text'],
+			// We are redefining the overall_status rather than translating it, so that:
+			// • the v6 overall_status performances does not degrades from the additional FROMs
+			// • the behavior does not change if we later add new statuses in the v7 one
+			'overall status': [
+				'Case',
+				[
+					'When',
+					[
+						'Or',
+						[
+							'In',
+							['ReferencedField', 'device', 'status'],
+							['EmbeddedText', 'Ordered'],
+							['EmbeddedText', 'Preparing'],
+						],
+						[
+							'And',
+							['Not', ['ReferencedField', 'device', 'is online']],
+							[
+								'Equals',
+								['ReferencedField', 'device', 'status'],
+								['EmbeddedText', 'Shipped'],
+							],
+						],
+					],
+					['ToLower', ['ReferencedField', 'device', 'status']],
```

Comment:
Should we also drop these non-used & already dropped HoD states, rather re-adding them?